assets/panel/debugger.properties

```diff
@@ -152,6 +152,30 @@ projectTextSearch.key=CmdOrCtrl+Shift+F
 # modal for searching functions in a file.
 functionSearch.key=CmdOrCtrl+Shift+O
 
+# LOCALIZATION NOTE (functionSearch.key): A key shortcut to toggle
+# breakpoints.
+toggleBreakpoint.key=CmdOrCtrl+B
+
+# LOCALIZATION NOTE (functionSearch.key): A key shortcut to toggle
+# the conditional breakpoint panel.
+toggleCondPanel.key=CmdOrCtrl+B
+
+# LOCALIZATION NOTE (functionSearch.key): A key shortcut to toggle
+# pause and resume.
+pauseOrResume.key=F8
```

Comment:
We probably don't need F8 in properties, we can just include the string directly